server/src/main/java/io/crate/analyze/CopyStatementSettings.java

```diff
@@ -35,6 +35,11 @@
     private CopyStatementSettings() {
     }
 
+    public static final Setting<String> PROTOCOL_SETTING = Setting.simpleString(
+        "protocol",
+        "https",
+        Setting.Property.Dynamic);
+
```

Comment:
If we declare the setting here as well it breaks the idea of moving them into the `CopyPlugin`.

For this pluggability to work `server` must not know anything about scheme specific settings.